include/links.php

```diff
@@ -28,6 +28,13 @@ class PLL_Links {
 	 */
 	public $links_model;
 
+	/**
+	 * Current language (used to filter the content).
+	 *
+	 * @var PLL_Language|null
+	 */
+	public $curlang;
+
```

Comment:
Because of this:
```log
 ------ ------------------------------------------------------ 
  Line   include/switcher.php                                  
 ------ ------------------------------------------------------ 
  69     Access to an undefined property PLL_Links::$curlang.  
 ------ ------------------------------------------------------
```

Reply:
Understood. We can remove it from `PLL_Frontend_Links` then.

Reply:
Done in [4878f20](https://github.com/polylang/polylang/pull/1143/commits/4878f204f07daed17e304f731767f03a9d6aed49).